DAO/project/DAO-contract/tests/utils/mod.rs

```diff
@@ -195,19 +198,16 @@ pub mod test_helpers {
         .await
         .unwrap();
 
-        let gov_token = GovToken::new(gov_token_id.to_string(), deployer_wallet.clone());
+        let gov_token = GovToken::new(gov_token_id.clone(), deployer_wallet.clone());
```

Comment:
The `new()` function now takes `Bech32ContractId` arguments instead of `String`. 

Actually, `clone()` is not needed here, since the `gov_token_id` is not used again in this scope. Removed this along with some other unnecessary `clone()`s.